test/integration/README.md

```diff
@@ -0,0 +1,34 @@
+## Conformance test duration log
+
+Design document link - https://quip-amazon.com/BoJLAC3IpIfW 
```

Comment:
This link doesn't work outside Amazon, we should only refer to documents in this repo or the public docs at AWS.